src/win/fs.c

```diff
@@ -1971,8 +1971,51 @@ INLINE static DWORD fs__stat_impl_from_path(WCHAR* path,
                        flags,
                        NULL);
 
-  if (handle == INVALID_HANDLE_VALUE)
-    return GetLastError();
+  if (handle == INVALID_HANDLE_VALUE) {
+    DWORD error = GetLastError();
+    int sys_error = uv_translate_sys_error(error);
+    WIN32_FIND_DATAW findFile_data;
+    HANDLE find_handle;
+    FILE_STAT_BASIC_INFORMATION stat_info;
+
+    if (sys_error != UV_EBUSY && sys_error != UV_EPERM) {
```

Comment:
Wouldn't it make more sense to check the native Windows error here?

Reply:
I used the translated errors because if a new error is mapped to these 2 in the future, it might be worth trying this `FindFristFileW` for the new ones. Someone can add a new map in the header file but can easily forget to add it here. 
However, if you want me to copy all those native Windows errors, I can do so.